Gemfile.lock

```diff
@@ -250,25 +245,25 @@ GEM
     method_source (0.9.2)
     mime-types (3.3)
       mime-types-data (~> 3.2015)
-    mime-types-data (3.2019.0904)
+    mime-types-data (3.2019.1009)
```

Comment:
### [mime-types-data](https://github.com/mime-types/mime-types-data/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.2019.0904 → 3.2019.1009

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **3** commits. See the full changes on [the compare page](https://github.com/mime-types/mime-types-data/compare/v3.2019.0904...v3.2019.1009).

These are the individual commits:
- (89275be) [Give pdf extension higher priority (#24)](https://github.com/mime-types/mime-types-data/commit/89275be165ac5282f59e2c92d1a06a17e50ebfc8)
- (feb9a9a) [Update 3.2019.1009](https://github.com/mime-types/mime-types-data/commit/feb9a9a954097a53edbfc2a33842e4d7017b3ef7)
- (f79bc2d) [Update 3.2019.1009](https://github.com/mime-types/mime-types-data/commit/f79bc2d8d4cfbbb0516a3250c2077c2f97b705ea)


</details>